FND/Agreements/Contracts.rdf

```diff
@@ -215,13 +213,26 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;UnilateralContract"/>
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;WrittenContract"/>
 		<rdfs:label>promissory note</rdfs:label>
-		<skos:definition>a promissory note is a written, signed, unconditional, and unsecured promise by one party (the maker or promisor) to another (the payee or promisee) that commits the maker to pay a specified sum on demand, or on a fixed or a determinable date</skos:definition>
-		<skos:editorialNote>Unlike a contract, a Promissory Note does not need to be signed by both parties. It is essentiually a promise from one party to the holder, of some good or benefit. Promissory notes would generally by fully fungible. These are modeled as a kind of contract but are essentially a kind of unilateral contract between the issuer and the holder, and some authorities might not see this as a contract at all. Cash is a kind of promissory note, with the issuer being a central bank.</skos:editorialNote>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/promissory-note.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>unconditional and unsecured promise by one party (the issuer or promisor) to another (the payee or promisee) that commits the issuer to pay a specified sum on demand, or on a fixed or a determinable date</skos:definition>
```

Comment:
Should there not be a restriction that there is exactly one ContractPrincipal?

Reply:
I'm not a contract attorney, so I don't know.  I almost made that change, but decided against it in case there is some situation in which you can have more than one.

Reply:
OK fair enough, but I'd like to see some examples of all this Contract/Party and subclasses/subproperties which end up being Instruments. This is after all the very core of FIBO.